dockers/docker-platform-monitor/supervisord.sh.j2

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Generate additional supervisord config file
+{% if not skip_sensors %}
+# If this platform has an lm-sensors config file, copy it to it's proper place
+# and start lm-sensors
+if [ -e /usr/share/sonic/platform/sensors.conf ]; then
+    mkdir -p /etc/sensors.d
+    /bin/cp -f /usr/share/sonic/platform/sensors.conf /etc/sensors.d/
+    /bin/cp -f /usr/share/sonic/templates/sensors.conf /etc/supervisord/conf.d/
+fi
+{% endif %}
+
+{% if not skip_fancontrol %}
+# If this platform has a fancontrol config file, copy it to it's proper place
+# and start fancontrol
+if [ -e /usr/share/sonic/platform/fancontrol ]; then
+    # Remove stale pid file if it exists
+    rm -f /var/run/fancontrol.pid
+
+    /bin/cp -f /usr/share/sonic/platform/fancontrol /etc/
+    /bin/cp -f /usr/share/sonic/templates/fancontrol.conf /etc/supervisord/conf.d/
```

Comment:
Why is fancontrol in the templates/ directory, as it is not a template?

Reply:
done